docs/integrations/evm.mdx

```diff
@@ -0,0 +1,165 @@
+     */
+    function consume_entropy(uint maxAge) public view returns (bytes memory) {
+        // Consumers can specify the maximum entropy age they are willing to accept
+        require(block.timestamp - last_update <= maxAge, "ENTROPY_TOO_OLD");
+
+        return entropy;
+    }
+}
+```
+Deployed contract example on goerli chain: [0x2c503cfba7178eb0ac2dc5df45279527d437657a](https://goerli.etherscan.io/address/0x2c503cfba7178eb0ac2dc5df45279527d437657a)
+
+### Job specification
+
+Now that the contract has been deployed, we can prepare the job script that will get executed by the processor to provision the entropy.
+
+The script below computes some random bytes securely and calls the function `receive_entropy` of the contract deployed above. The consumer can also specify the gas limit and maximum costs per gas unit.
+
+<RunnableCode>
+
+```javascript title="JobLogic.js"
+// Destination contract address
+const destination = "<0x evm_contract_address>";
+// Generate entropy
+const entropy = _STD_.random.generateSecureRandomHex();
+// Fulfill entropy
+_STD_.chains.ethereum.fulfill(
+    "https://rpc.ankr.com/eth_goerli",  // RPC
+    destination,                        // Destination contract address
+    entropy,                            // Payload
+    // Transaction parameters
+    {
+        methodSignature: "receive_entropy(bytes)",
+        gasLimit: "9000000",
+        maxFeePerGas: "255000000000",
+        maxPriorityFeePerGas: "2550000000",
+    },
+    // Success callback
+    (opHash) => {
+        print("Succeeded: " + opHash)
+    },
+    // Error callback
+    (err) => {
+        print("Failed: " + err)
+    },
+);
+```
+</RunnableCode>
+<div class="padding-vert--md"></div>
+
+Once you finish implementing the job script, you will have to publish it to IPFS. If you do not know what IPFS is, check out this link https://docs.ipfs.tech/concepts/what-is-ipfs.
+
+You can use [Pinata](https://app.pinata.cloud) to pin the script to the IPFS network.
+
+IPFS link example: [ipfs://QmPcSHumW7DFzBZ3VC9Yc8QuxnaJbfNB7BGKGrNDdn4S3w](https://cloudflare-ipfs.com/ipfs/QmPcSHumW7DFzBZ3VC9Yc8QuxnaJbfNB7BGKGrNDdn4S3w)
+
+### Job registration
+
+Currently, the job registration process requires some technical knowledge which may be difficult to understand by non developer users. We expect the simplify the process in a future release of https://console.acurast.com.
+
+For now, you can register a job by going to [https://polkadot.js.org](https://polkadot.js.org/apps/?rpc=wss://wss.collator-1.acurast.papers.tech/#/extrinsics) and following the steps below.
+
+1. In the extrinsics view, you will need to select extrinsic **`register`** from **`acurast`** pallet;
+2. Then you need to specify the IPFS script provisioned in [job specification section](#job-specification);
+
+<ThemedImage
+  sources={{
+    light: useBaseUrl("/guide/job-registration/step1.png"),
+    dark: useBaseUrl("/guide/job-registration/step1.png"),
```

Comment:
Where are these assets located?
Cc https://github.com/w3f/Grant-Milestone-Delivery/pull/606

Reply:
The assets are located here:

![image](https://user-images.githubusercontent.com/22591718/223635657-11298f15-ea3d-45b8-a30f-2c68b67cf311.png)

Reply:
Fair enough 😂
Thanks for enlightening me 👍